components/kyma-environment-broker/internal/broker/testdata/aws-schema-additional-params.json

```diff
@@ -14,7 +14,7 @@
     "administrators": {
       "description": "Specifies the list of runtime administrators",
       "items": {
-          "type": "string"
+        "type": "string"
```

Comment:
this unrelated whitespace diff appeared after getting all `testdata` fixtures in sync with `/catalog` endpoint output, now the indent is consistent 2 spaces per each level.